openmoc/docstring.i

```diff
@@ -2821,61 +2802,40 @@ the number of threads per block
 %feature("docstring") GPUSolver::computeResidual "
 computeResidual(residualType res_type) -> double  
 
-Computes the residual between source/flux iterations.  
+Computes the residual between successive flux/source iterations.  
 
 Parameters
 ----------
 * res_type :  
-    the type of residuals to compute (SCALAR_FLUX, FISSION_SOURCE, TOTAL_SOURCE)  
+    the type of residual (FLUX, FISSION_SOURCE, TOTAL_SOURCE)  
```

Comment:
Oh I see, it is pulling this comment from "Solver.h" where it was indeed incorrect. Good thing to note for the future - "doxy2swig.py" is smart enough to pull comments from parent classes and inject them as docstrings into their children.